<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testModule</class>
 <widget class="QDialog" name="testModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;C++ &lt;/span&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#d3272a;&quot;&gt;Test&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Bestätigen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Eine Variable mit dem Namen &amp;quot;x&amp;quot; soll als Typ Integer deklariert&lt;/p&gt;&lt;p&gt;und initialisiert werden. Der Wert soll 5 betragen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>85</y>
     <width>16</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>92</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>int</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>80</y>
     <width>20</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Eine Zeichenkette mit dem Namen &amp;quot;message&amp;quot; soll&lt;/p&gt;&lt;p&gt;mit dem Wert &amp;quot;Dies ist ein Test&amp;quot;&lt;/p&gt;&lt;p&gt;deklariert und initialisiert werden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>        message   = &quot;                                &quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb3">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>230</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb4">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>230</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Wo wird eine Variable verwendet? 
Welche der drei Aussagen treffen zu?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Innerhalb einer Methode und einer Klasse.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Innerhalb einer Methode außerhalb einer Klasse.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Außerhalb einer Methode innerhalb einer Klasse.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
